docs/user-guide/replicasets.md

```diff
@@ -0,0 +1,91 @@
+---
+---
+
+* TOC
+{:toc}
+
+# Replica Set
+
+## What is a _Replica Set_?
+
+Replica Set is the next-generation Replication Controller. The only difference between a
+_Replica Set_ and a [_Replication Controller_](replication-controller.md) right now is
+the selector support. Replica Set supports the new set-based selector requirements as
+described in the [labels user guide](labels.md#label-selectors) whereas a Replication
+Controller only supports equality-based selector requirements.
+
+Most [`kubectl`](kubectl/kubectl.md) commands that support Replication Controllers also
+support Replica Sets. Commands that support Replica Sets are:
+[`scale`](kubectl/kubectl_scale.md), [`autoscale`](kubectl/kubectl_autoscale.md),
```

Comment:
Good point.  It might be simpler to just say the things that aren't supported because they are Replication Controller specific.  Good catch @janetkuo.